src/styles/utils/normalize.css

```diff
@@ -0,0 +1,8 @@
+.header,
+.header__list,
+.card__stars-container,
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
```

Comment:
I think this styles isn't related to normalize.css

Here is a show breif about normalize.css:
**Normalize.css** - as the name suggests, it normalizes styles in the browsers for their user agents, i.e. makes them the same across all browsers due to the reason by default they're slightly different.

Example: `<h1>` tag inside `<section>` by default Google Chrome will make smaller than the "expected" size of `<h1> `tag. Microsoft Edge on the other hand is making the "expected" size of `<h1>` tag. Normalize.css will make it consistent.